clcache/__main__.py

```diff
@@ -118,6 +118,19 @@ def normalizeBaseDir(baseDir):
         return None
 
 
+def caseInsensitiveReplace(data, old, new):
+    dataLower = data.lower()
+    oldLower = old.lower()
+    output = b''
+    index = 0
+    while True:
+        nextIndex = dataLower.find(oldLower, index)
+        if nextIndex < 0:
+            return output + data[index:]
+        output += data[index:nextIndex] + new
+        index = nextIndex + len(old)
+
+
```

Comment:
Rightly or wrongly, for simple things I personally like to avoid regex for performance reasons. But I'm a beginner in Python so if you prefer regex here I'm happy to oblige.